homeassistant/components/buienradar/strings.json

```diff
@@ -0,0 +1,35 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Buienradar",
```

Comment:
"Buienradar" is a brand name, it doesn't need to be translated.
Applies to other occurrences in this file as well.